barrantes_sofiaanne/index.css

```diff
@@ -0,0 +1,209 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, sans-serif;
+}
+
+.split-right {
+    height: 100%;
+    display: flex; 
+    background-color: #ffd1dc; 
+    align-items:center; 
+    padding: 20px;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.slide { 
+    max-width: auto;
+    margin-bottom: 20px; 
+    padding: 20px;
+    background-color: white;
+    border: 10px solid white;
+    box-sizing: border-box; 
+}
+
+.slide h2 {
+    margin-top: 0; 
+}
+
+#name {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+    max-width: 800px;
+    background-color: rgba(255, 255, 255, 0.8);
+    padding: 50px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+}
+
+.split-left {
+    height: 100vh; 
+    flex: 1;
+    display: flex; 
+    justify-content: center;
+    background-image: url("https://f8n-production-collection-assets.imgix.net/0x9937D9A7205238834F6aE86D5CbbE60a887cFb6F/1/nft.png?auto=format%2Ccompress&q=70&h=1200&w=1200");
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+ul {
+    padding-left: 20px;
+    margin: 0; 
+}
+
+a {
+    text-decoration: none;
+    color: #007bff;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+.img {
+    flex: 1;
+    max-width: 100%;
+    height: auto;
+}
+
+.section-container {
+    border: 2px solid white;
+    padding: 5px;
+    border-radius: 10px;
+    width: 100%;
+    box-sizing: border-box;
+}
+
+.gallery {
+    text-align: center; 
+    background-color: white ;
+}
+
+.gallery-item {
+    display: inline-block; 
+    margin: 10px; 
+    width: 175px; 
+    box-sizing: border-box; 
+}
+
+.gallery-item img {
+    width: 100%; 
+    height: auto; 
+    max-width: 150px; 
+}
+
+.email {
+    grid-area: email;
+}
+
+.Name {
+    grid-area: Name;
+}
+
+.subject {
+    grid-area: subject;
+}
+
+.age {
+    grid-area: age;
+}
+
+.message {
+    grid-area: message;
+}
+
+.submit {
+    grid-area: submit;
+    text-align: center;
+}
+
+.copy {
+    grid-area: copy;
+}
+
+.email, .Name, .subject, .message, .age, .copy {
+    border:3px solid white;
+    padding: 20px;
+    text-align:start;
+}
```

Comment:
make it  

`.email,
.Name,
.subject,
.message,
.age,
.copy {
  border: 3px solid white;
  padding: 20px;
  text-align: start;
}`

to become more readable